src/login/logind-session.c

```diff
@@ -1163,12 +1158,9 @@ static void session_restore_vt(Session *s) {
         if (vt < 0)
                 return;
 
-        (void) ioctl(vt, KDSETMODE, KD_TEXT);
-
-        (void) vt_reset_keyboard(vt);
-
-        (void) ioctl(vt, VT_SETMODE, &mode);
-        (void) fchown(vt, 0, (gid_t) -1);
+        r = vt_restore(vt);
+        if (r < 0)
+                log_error_errno(r, "Failed to restore VT: %m");
```

Comment:
if we ignore the errors then we should only log at WARNING level, and also say that we ignore this.